srcalc/src/main/java/gov/va/med/srcalc/domain/calculation/ValueRetriever.java

```diff
@@ -267,6 +269,13 @@ public void execute(final Patient patient, final VariableEntry variableEntry, fi
     {
     }
     
+    /**
+     * Attempt to retrieve any reference information for this instance. Do nothing if
```

Comment:
"Do nothing" is not quite accurate: what _does_ this method do if there is no associated reference info?